data_scraper.py

```diff
@@ -173,7 +173,7 @@ def _data_cleansing(self, df):
         )
 
         # get the 'SK' or 'LK' for Stadtkreis (city county) or Landkreis (rural county)
-        df["county"] = df["county"].str.slice(start=2)
```

Comment:
Shouldn't the target column then be 'kind_of_county'?

Reply:
A the columns are renamed afterwards

Reply:
The renaming is done afterwards. The thing is, that the data sets contains the name of the Landkreis twice. The 'county' column with a 'SK' or 'LK' prefix. I left the 'SK' and 'LK' and then renamed it.